docs/zh/latest/admin-api.md

```diff
@@ -927,6 +927,7 @@ Content-Type: text/plain
 | ----------- | ----------------------------------- | ---------- | ------------- |
 | GET         | /apisix/admin/plugins/list          | 无         | 获取资源列表  |
 | GET         | /apisix/admin/plugins/{plugin_name} | 无         | 获取资源      |
+| GET         | /apisix/admin/plugins?all=true      | 无         | 获取所有插件的所有属性|
```

Comment:
I noticed that we do not have `/apisix/admin/plugins/list ` and `/apisix/admin/plugins/{plugin_name} ` in our en docs, and it seems that we don't have this two APIs, so do we need to remove them?

Reply:
Look like it is an issue with the en docs. I can confirm these two APIs exist.